modules/book.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-mutable-exports */
+import {
+  container, header, form, titletext, contactpage, listTab, addingTab, contactTab,
+} from '../index.js';
+import { error } from './error.js';
+import { inputSave } from './localstorage.js';
+// import {bookarticles,directive,bookMessage} from './directive.js';
+// create a single book template
+export const inputTitle = document.querySelector('.title');
+export const inputAuthor = document.querySelector('.author');
+export const addButton = document.querySelector('.add');
+
+// create showcase for books
+export const bookDisplay = document.createElement('div');
+bookDisplay.classList.add('bookDisplay');
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+let bookHolder = [];
+
+export const addBook = () => {
+  const titleData = inputTitle.value;
+  const authorData = inputAuthor.value;
+
+  const newBook = new Book(titleData, authorData);
+
+  bookHolder.push(newBook);
+
+  // store the updated array in local storage
+  localStorage.setItem('storedBooks', JSON.stringify(bookHolder));
+
+  // clean the input
+  inputTitle.value = '';
+  inputAuthor.value = '';
+  inputSave();
+};
+
+export const deleteBook = (event) => {
+  const button = event.target;
+  const instance = button.parentNode;
+  instance.remove();
+  const index = button.getAttribute('data-index');
+  // bookHolder.splice(index, 1);
```

Comment:
- [ ] Please remove the commented code. The commented code makes your code messy and may make your code hard to maintain 👍.